### PR TITLE
presigned: Input expires should be time.Duration

### LINKED ISSUE #300: All arguments for all functions should have native types
For example in Golang in PresignedGetObject

```
func (a API) PresignedGetObject(bucket, object string, expires time.Duration) (string, error) {
```

Expires argument is time.Duration which is a native type under Golang ecosystem, we should make sure that wherever possible we should take native types.

### PR DESCRIPTION
**Pull request by harshavardhana**
Fixes #300

### CODE REVIEW

**Comment by harshavardhana** [comment]
@balamurugana ?

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,6 @@ language: java
 sudo: false
 jdk:
 - oraclejdk8
-- oraclejdk7
-- openjdk7
```
Comment (balamurugana) [inline .travis.yml]: does it mean we don't support only oraclejdk8?

File: src/main/java/io/minio/MinioClient.java [hunk 2]
```diff
@@ -1015,11 +1021,12 @@ public String presignedPutObject(String bucketName, String objectName, Integer e
    * @throws NoSuchAlgorithmException upon requested algorithm was not found during signature calculation
    * @throws InvalidExpiresRangeException upon input expires is out of range
    */
-  public String presignedPutObject(String bucketName, String objectName)
+  public String presignedDEFAULT_EXPIRY_TIMEPutObject(String bucketName, String objectName)
```
Comment (balamurugana) [inline src/main/java/io/minio/MinioClient.java]: looks like a typo or wrong copy/paste :)

**Comment by harshavardhana** [comment]
We will stick to JDK7 for now.

**Pull request closed without merging**